data/mods/blazemod/recipes/vortex_recipes.json

```diff
@@ -63,19 +63,6 @@
       [ [ "vortex_stone", 1 ] ]
     ]
   },
```

Comment:
The spiral_matrix is actually something belonging to the diamond group, renamed to diamond_matrix in the diamond group to avoid this confusion.